mne/coreg.py

```diff
@@ -39,6 +39,7 @@
 fid_fname = pformat(bem_fname, name='fiducials')
 fid_fname_general = os.path.join(bem_dirname, "{head}-fiducials.fif")
 src_fname = os.path.join(bem_dirname, '{subject}-{spacing}-src.fif')
+high_res_head_fname = os.path.join(subject_dirname, 'surf', 'lh.seghead')
```

Comment:
@agramfort `lh.seghead` is produced by FreeSurfer's `mkheadsurf` (the first step of `$ mne make_scalp_surfaces`) and seems to contain the same information as `head-dense-bem.fif` (as in `np.allclose`). My next plan was to generate `lh.seghead` when it is not present with a call to FreeSurfer. Is there a reason to create an additional file with the same information? If there is a use case for modifying `head-dense.fif` we could look for `head-dense.fif` before `lh.seghead`?

@dengemann that is the intention behind this PR